docs/io/unified.rst

```diff
@@ -230,167 +230,177 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
-  in the format of their choosing.
-
-      >>> from astropy.time import Time
-      >>> from astropy.table import Table
-      >>> from astropy.coordinates import EarthLocation
-      >>> t = Table()
-      >>> t['a'] = Time([100.0, 200.0], scale='tt', format='mjd',
-      ...               location=EarthLocation(-2446354, 4237210, 4077985, unit='m'))
-      >>> t.write('my_table.fits', overwrite=True)
-      >>> tm = Table.read('my_table.fits')
-      >>> tm['a']
-      <Column name='a' dtype='float64' length=2>
-      100.0
-      200.0
-      >>> tm['a'] == t['a'].value
-      array([ True,  True], dtype=bool)
-
-  .. note::
-
-     The Astropy `~astropy.time.Time` object does not precisely map to the FITS Time standard.
-
-     * FORMAT
-
-       The FITS format considers only three formats, ISO-8601, JD and MJD.
-       Astropy Time allows for many other formats like ``unix`` or ``cxcsec``
-       for representing the values.
-
-     * LOCATION
-
-       In Astropy Time, location can be an array which is broadcastable to the
-       Time values. In the FITS standard, location is a scalar expressed via
-       keywords.
-
-     Hence, ``format`` and ``vector location`` are not stored, due to which
-     complete round-tripping of ``Time`` is not supported.
-
-     Reading FITS files with time coordinate columns which are not written by
-     Astropy, is not supported and hence, will fail if you try to do so.
-
-  The ability to read in the various global time informational keywords,
-  like the ``DATE``, ``DATE-*`` ISO-8601 datetime strings and the ``MJD-*``
-  mjd values as ``Time`` objects in the Table ``meta`` has also been provided.
-  For more details regarding the FITS time paper and the implementation,
-  refer to :ref:`fits_time_column`.
+object attributes beyond the standard Column attributes like ``format`` or
+``description``. Abiding by the rules set by the FITS standard requires mapping
+of these data components and object attributes to the appropriate FITS table
+columns and keywords.  Thus a well defined protocol has been developed to allow
+the storage of these mixin columns in FITS while allowing the object to
+"round-trip" through the file with no loss of data or attributes.
+
+Quantity
+~~~~~~~~
+
+A `~astropy.units.Quantity` mixin column in a `~astropy.table.QTable` is
+represented in a FITS table using the ``TUNITn`` FITS column keyword to
+incorporate the unit attribute of Quantity.
+
+    >>> from astropy.table import QTable
+    >>> import astropy.units as u
+    >>> t = QTable([[1, 2] * u.angstrom)])
+    >>> t.write('my_table.fits', overwrite=True)
+    >>> qt = QTable.read('my_table.fits')
+    >>> qt
+    <QTable length=2>
+      col0
+    Angstrom
+    float64
+    --------
+         1.0
+         2.0
+
+Time
+~~~~
+
+By default, a `~astropy.time.Time` mixin column within `~astropy.table.Table`
+or `~astropy.table.QTable` will be written to FITS as a standard column using
```

Comment:
within ``a`` `~astropy.table.Table`
Extremely minor change.

Also should ``column`` be ``Column``?